cmd/generate/config/rules/vault.go

```diff
@@ -0,0 +1,38 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func VaultServiceToken() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Vault Service Token",
+		RuleID:      "vault-service-token",
+		Regex:       generateUniqueTokenRegex(`hvs\.[a-z0-9_-]{90,}`),
+		Keywords:    []string{"hvs"},
+	}
+
+	// validate
+	tps := []string{
+		generateSampleSecret("vault", "hvs."+secrets.NewSecret(alphaNumericExtendedShort("90"))),
+	}
+	return validate(r, tps, nil)
+}
+
+func VaultBatchToken() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Vault Batch Token",
+		RuleID:      "vault-batch-token",
+		Regex:       generateUniqueTokenRegex(`hvb\.[a-z0-9_-]{138,}`),
```

Comment:
Same concern ashttps://github.com/zricethezav/gitleaks/pull/919/files#r931665897

Reply:
This is based on:
- [Vault example](https://learn.hashicorp.com/tutorials/vault/batch-tokens): hvb.AAAAAQJyBEVE-vTWUrg0hcoIPuvKjjNxXXZ5MfsYVg2gJ0fGZpVi0IGTFfh4TqsoQIWaocNRXD1qzGXvhIHWJBM_rWU9YJY8sXOYVy_s1JAHasXJwGmZ_fBLJfSG6aCwQkCGwtAhYw
- https://gitlab.com/gitlab-org/security-products/analyzers/secrets/-/merge_requests/133/diffs

I also generated some tokens for two different non-enterprise clusters:
```
hvb.AAAAAgLPlfPCLcUODt30I2ZQZbzjki5uWP_rNKh-Mr18Goo-z_UlKeOtQzV8h-PJVGmW6bjK4jsOAkkYL5EZLNgJnXxjVIMamcIufw16OtBfmFWLesYPMMzrUOaTEeb-rBo2MfjRZLm0_N-OU56TgIrgwTtxnhJn5Rc5ohb-jBWggdJ7WpQ4kGnMSSb_2PvnHUAXWu3LGGnXCbOXeMDn
hvb.AAAAAgL2FY1LChaErHWlrq_wWuPeaebP7fmpgShy4L2RoCRoYoLjxrFrOZp7FCASNepgpJW_cyHmJqhuFcHCRUZagoiOlmhDP7812-wFm2tR0rh8IK7SloJU67sFrBI7rJ0epXWNZg-Zn7rslqBRaM0LSRLyDuuLUTSOzLILlZLgrdQvHRc4PUjAWsAPfMVRQD2ZPM88-FYTwkpgq-FV

hvb.AAAAAQLEJtaK0mb4FRPplZ4ak7QVHKbOQqmCyJEzcNbgFPO0YPNfo4WcZNp22oXFl8L5D8XNzGp0nYDRn7Qo9iUGJUmiNKmBNUE51sIJcm8YQ_5fMHRD5WWW2EZCaxIrYPYvf2iXVpCJLWbQziDhiPYcwB9f_C_ZQ9ny6km9DR12TccrL9GsHWx5A9GNGMAAbz1ZsBxuEyw9HIuzASnV
hvb.AAAAAQI3ZCdM2lGQGUYQwNG4Wgc-TCw1KqQ_ShEBp4hcIeYfCIBOl_YTOYyI6T_zgJzmOWsac0XdQ_chMh4oo-9yDfuD1PURICsUorb_-_lRWpiAWbN5G6JaOUclXfw2tIHjVi0c7i_6gsXkyFcA7RDCNo9Z-ugNlO4PbE5-3mlzhx9jA9tSKgejsK5Vk1vSBtpo82gZiEV1-fQwS1Qt
```

So I guess it could be `{138,212}` (or just {212} based on our tokens) but again - I can't confirm if it works with all Vault credentials 🤷‍♂️

Reply:
`{138,212}` is better than unbounded. Let's go with that for now

Reply:
Sounds good - I updated the PR.